database/3 reports.sql

```diff
@@ -0,0 +1,46 @@
+use DpCampDB
+go
+
+-- 1) Leah Burstein is searching for her twin. She wants to know in which camp she is in now and how many days she has been there for.
+-- SM I'm doing december 31 1947 because you say current year is 1947 but I have no idea the date.
+select d.DpFirstName, d.DpLastName, d.Camp, NumOfDaysInCamp = datediff(day,d.ArrivalDate,'19471231')
```

Comment:
Well, this works for now but when another sister arrives it won't work. Since she's searching for her twin, make sure that the DOB is the same.

Reply:
Included it.

Reply:
great!